includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -2560,4 +2561,48 @@ private function get_fingerprint_metadata(): array {
 
 		return $customer_fingerprint_metadata;
 	}
+
+	/**
+	 * List authorizations
+	 *
+	 * @param int    $page       The requested page.
+	 * @param int    $page_size  The size of the requested page.
+	 * @param string $sort       The column to be used for sorting.
+	 * @param string $direction  The sorting direction.
+	 *
+	 * @return array
+	 * @throws API_Exception - Exception thrown on request failure.
+	 */
+	public function list_authorizations( $page = 0, $page_size = 25, $sort = 'created', $direction = 'desc' ) {
```

Comment:
Can we add types declarations here, please?

Reply:
Thank you for reminding us of this. Fixed in f452270d